test/rules/overridden_fields.dart

```diff
@@ -162,3 +162,7 @@ class StaticsNo {
 class VerifyStatic extends StaticsNo {
   static int a;
 }
+
+mixin M on A1 {
```

Comment:
Maybe also positive / negative tests for the lint?

Reply:
Done